Add Request Notification Section

naman-crabnebula opened the pull request:
#### Changes
- New or updated content

#### Description
- Closes #1483 
- Add more comprehensive guide for the Tauri Notification Plugin
- Adding sections like `Requesting Permissions`.
- It is initially a draft PR and will grow as I write the content. I will ask for review once it is completed. 
- Feel free to make suggestions!

naman-crabnebula committed:
Add Request Notification Section

vasfvitor commented:
> Feel free to make suggestions!

since you mentioned, it's not a suggestion but a reminder to add the compatibility and permission list section

naman-crabnebula commented:
> add the compatibility and permission list section

I already added the Permission section, but have no idea about compatibility one. I will ping you on discord regarding this. 

Thank You

naman-crabnebula committed:
Add supported platforms

Sir-Thom committed:
docs: Adding RPM doc (#2316)

renovate[bot] committed:
fix(deps): update dependency typescript to v5.5.3 (#2353)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

simonhyll committed:
feat: edit page links open vscode during development

simonhyll committed:
fix: article ordering in learn category

lucasfernog committed:
blog: announce Tauri 1.7 (#2354)

simonhyll committed:
fix: use intended expressiveCode style override

simonhyll committed:
fix: netlify developer frame csp

renovate[bot] committed:
fix(deps): update dependency starlight-links-validator to v0.9.1 (#2358)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency shiki to v1.10.1 (#2357)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency starlight-blog to ^0.10.0 (#2359)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency astro to v4.11.4 (#2360)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

Nabellaleen committed:
fix: fix cargo command in create-project.mdx (#2361)

simonhyll committed:
Revert "fix(deps): update dependency astro to v4.11.4" (#2365)

FabianLars committed:
ci: update workflow versions

FabianLars committed:
ci: Remove pnpm version from workflow

RoseBlume committed:
docs: Add Github Actions Docs (#2313)

Co-authored-by: Breno A <git@breno.tech>
Co-authored-by: FabianLars <fabianlars@fabianlars.de>

renovate[bot] committed:
fix(deps): update dependency starlight-blog to ^0.11.0 (#2368)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency astro to v4.11.5 (#2362)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

zrq8 committed:
docs: fix typo on the package name (#2369)

FabianLars committed:
docs: fix nextjs frontendDist

renovate[bot] committed:
fix(deps): update dependency @types/node to v20.14.10 (#2371)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency typedoc-plugin-mdn-links to v3.2.3 (#2375)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/tauri digest to 24aaf0f (#2374)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency shiki to v1.10.2 (#2376)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency shiki to v1.10.3 (#2378)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

RomainMazB committed:
docs: Fix environment variables link (#2380)

charlesschaefer committed:
docs: Adding an example of how to trigger a notification from rust code. (#2381)

Co-authored-by: FabianLars <fabianlars@fabianlars.de>

AurimarL committed:
fix nextjs beforeBuildCommand (#2383)

renovate[bot] committed:
fix(deps): update dependency @lorenzo_lewis/starlight-utils to v0.1.2 (#2382)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

GuoJikun committed:
i18(zh-cn): Synchronizing content (#2352)

renovate[bot] committed:
chore(deps): update pnpm to v9.5.0 (#2379)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/plugins-workspace digest to ac9a25c (#2373)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/awesome-tauri digest to e2671d7 (#2372)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Paul Valladares <85648028+dreyfus92@users.noreply.github.com>

simonhyll committed:
fix: trailing slashes always on

simonhyll committed:
fix: disable trailing slash enforcement

amrbashir committed:
feat: android signing guide (#2337)

Co-authored-by: Simon Hyll <hyllsimon@gmail.com>

jbolda committed:
Add Description To Store Plugin (#2384)

Co-authored-by: Simon Hyll <hyllsimon@gmail.com>

jbolda committed:
[i18nIgnore] Import Examples When `"withGlobalTauri": true` (#2385)

danny900714 committed:
Fixes typo in `scope.mdx` (#2390)

renovate[bot] committed:
fix(deps): update dependency sass to v1.77.7 (#2386)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

danny900714 committed:
Change the description of append to prepend for `tauri-plugin-` (#2389)

KikkiZ committed:
i18n(zh-cn): Update the translation of from-tauri-1 (#2343)

Co-authored-by: FabianLars <fabianlars@fabianlars.de>

murnifine committed:
docs(upload): Fix progress handler syntax(#2392)

Co-authored-by: Fabian-Lars <fabianlars@fabianlars.de>

flapili committed:
Fix configuration handling in Plugin's doc code snippet (#2394)

GuoJikun committed:
i18(zh-cn): translate `distribute/rpm.mdx` (#2393)

Co-authored-by: DK Liao <dklassic@gmail.com>

renovate[bot] committed:
fix(deps): update dependency sass to v1.77.8 (#2396)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

GuoJikun committed:
update `distribute/snapcraft.mdx` (#2397)

renovate[bot] committed:
fix(deps): update dependency prettier to v3.3.3 (#2407)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency typedoc-plugin-mdn-links to v3.2.4 (#2408)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

Zercerium committed:
fix broken link in store.mdx to state-management (#2409)

renovate[bot] committed:
chore(deps): update packages/awesome-tauri digest to cebe745 (#2401)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Paul Valladares <85648028+dreyfus92@users.noreply.github.com>

StepanZagray committed:
docs: Update window customization guide (#2411)

Co-authored-by: Fabian-Lars <fabianlars@fabianlars.de>

fu050409 committed:
docs(ci): update tauri dependencies in webdriver workflow example (#2399)

Co-authored-by: Fabian-Lars <fabianlars@fabianlars.de>

GuoJikun committed:
[i18nIgnore] Fix JS API Reference link are broken.  (#2415)

renovate[bot] committed:
fix(deps): update dependency prettier-plugin-astro to v0.14.1 (#2418)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency @types/node to v20.14.11 (#2420)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency astro-d2 to ^0.4.0 (#2419)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

GuoJikun committed:
[i18nIgnore] Fix broken links (#2416)

renovate[bot] committed:
fix(deps): update dependency semver to v7.6.3 (#2421)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency astro to v4.11.6 (#2423)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update astro monorepo (#2429)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/plugins-workspace digest to 77ee644 (#2402)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Paul Valladares <85648028+dreyfus92@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/tauri digest to 88552d6 (#2404)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency astro to v4.12.1 (#2433)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

StepanZagray committed:
Update calling rust from frontend guide (#2426)

Co-authored-by: Vitor Ayres <gitme@virtuaires.com.br>

renovate[bot] committed:
fix(deps): update dependency astro to v4.12.2 (#2435)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/tauri digest to 8b3e6ec (#2438)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/awesome-tauri digest to 3b736f8 (#2436)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/plugins-workspace digest to 93bdb6d (#2437)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

GuoJikun committed:
update `plugin/deep-link.mdx` (#2432)

Co-authored-by: Paul Valladares <85648028+dreyfus92@users.noreply.github.com>

simonhyll committed:
feat: automations for project management (#2434)

JacobStephenson97 committed:
[i18n-ignore] fixed typo in migrate (#2443)

GuoJikun committed:
i18n(zh-cn): Translate `start/frontend/laptos.mdx` (#2431)

Co-authored-by: DK Liao <dklassic@gmail.com>

GuoJikun committed:
[i18n-ignore] update `.github/PULL_REQUEST_TEMPLATE.md` (#2446)

GuoJikun committed:
i18n(zh-cn): translate `distribute/snapcraft.mdx` (#2398)

Co-authored-by: DK Liao <dklassic@gmail.com>

renovate[bot] committed:
fix(deps): update dependency jsdom to v24.1.1 (#2442)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency typedoc-plugin-mdn-links to v3.2.5 (#2441)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Paul Valladares <85648028+dreyfus92@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency shiki to v1.11.0 (#2445)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

simonhyll committed:
fix: use tauri bot token

simonhyll committed:
fix: use tauri bot token

simonhyll committed:
fix: use tauri bot token

simonhyll committed:
fix: skip auth with the cli

simonhyll committed:
Update status.yml

gabeins committed:
docs(sql): add `preload` config example for migrations (#2417)

FabianLars committed:
ci: Update assign.yml

FabianLars committed:
ci: Update status.yml

renovate[bot] committed:
fix(deps): update dependency starlight-blog to v0.11.1 (#2452)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Fabian-Lars <fabianlars@fabianlars.de>

tillmann-crabnebula committed:
Add Explanation on Dev Server Security (#2449)

This adds a section to the lifecycle security explaining how development servers for the frontend can be insecure and that the default vanilla dev server has no authentication or encryption.

It is also mentioned in the develop section.

renovate[bot] committed:
fix(deps): update dependency typescript to v5.5.4 (#2455)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

GuoJikun committed:
Add flag `i18nReady` (#2454)

Legend-Master committed:
docs: Add vscppdbg guide (#2400)

Co-authored-by: FabianLars <fabianlars@fabianlars.de>

renovate[bot] committed:
fix(deps): update dependency shiki to v1.11.1 (#2456)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency @types/node to v20.14.12 (#2457)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

FabianLars committed:
docs: Update snap instructions to not include src and build files in resulting file.

RoseBlume committed:
docs: Update Distribute Overview (#2403)

Co-authored-by: Breno A <git@breno.tech>
Co-authored-by: FabianLars <fabianlars@fabianlars.de>

FabianLars committed:
docs: Use gnome extension in snapcraft template

renovate[bot] committed:
fix(deps): update dependency shiki to v1.11.2 (#2461)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency shiki to v1.12.0 (#2462)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

KikkiZ committed:
i18n(zh-cn): translate develop/Plugin/resources.mdx (#2410)

KikkiZ committed:
i18n(zh-cn): translate /start/create-project.mdx (#2405)

chippers committed:
fix typo in develop mobile plugin (#2474)

vasfvitor committed:
[i18n-ignore] increase js-api toc depth to 5 (#2473)

lucasfernog committed:
feat: update configuration files documentation (#2467)

Co-authored-by: Vitor Ayres <gitkey@virtuaires.com.br>
Co-authored-by: Tillmann <28728469+tweidinger@users.noreply.github.com>
Co-authored-by: Fabian-Lars <fabianlars@fabianlars.de>

lucasfernog committed:
feat: fill calling-frontend documentation (#2468)

Co-authored-by: Tillmann <28728469+tweidinger@users.noreply.github.com>

lucasfernog committed:
feat: expand develop section, closes #1955 (#2463)

Co-authored-by: FabianLars <fabianlars@fabianlars.de>

renovate[bot] committed:
fix(deps): update dependency starlight-links-validator to ^0.10.0 (#2476)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency astro to v4.12.3 (#2477)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency @types/node to v20.14.13 (#2470)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

vasfvitor committed:
[i18n-ignore] show version on release cards (#2471)

Co-authored-by: Paul Valladares <85648028+dreyfus92@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/awesome-tauri digest to e246464 (#2464)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Paul Valladares <85648028+dreyfus92@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/plugins-workspace digest to e847ced (#2465)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Paul Valladares <85648028+dreyfus92@users.noreply.github.com>

Rigidity committed:
feat: State Management page (#2287)

Co-authored-by: Tillmann <28728469+tweidinger@users.noreply.github.com>

tweidinger committed:
Clarify the new default permissions and how to use plugin permissions (#2481)

renovate[bot] committed:
fix(deps): update dependency typedoc-plugin-mdn-links to v3.2.6 (#2469)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

FabianLars committed:
chore: Remove `packageManager` field and solely rely on `engines.pnpm`

renovate[bot] committed:
fix(deps): update dependency astro to v4.13.0 (#2482)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

FabianLars committed:
Revert "chore: Remove `packageManager` field and solely rely on `engines.pnpm`"

This reverts commit e936ec650caa95879e92f487df3ed4d8ac51016b.

renovate[bot] committed:
chore(deps): update pnpm to v9.6.0 (#2485)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency shiki to v1.12.1 (#2483)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency starlight-links-validator to v0.10.1 (#2484)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

GuoJikun committed:
i18n(zh-cn): Synchronizing content (#2453)

GuoJikun committed:
i18n(zh-cn): Translate `distribute/apk-sign.mdx` (#2448)

GuoJikun committed:
i18n(zh-cn): Translate `develop/sidecar.mdx` (#2458)

Co-authored-by: Embers-of-the-Fire <stellarishs@163.com>

renovate[bot] committed:
fix(deps): update dependency @types/node to v20.14.14 (#2487)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency astro to v4.13.1 (#2488)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

tweidinger committed:
Migration Guide From V2 Beta to Release Candidate (#2480)

Co-authored-by: Lucas Nogueira <lucas@tauri.app>

tweidinger committed:
feat: Add 2.0 Release Candidate Announcement (#2478)

Co-authored-by: Lucas Nogueira <lucas@tauri.app>

lucasfernog committed:
feat: update docs for v2 RC (#2489)

renovate[bot] committed:
chore(deps): update packages/plugins-workspace digest to ad03b8e (#2490)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

chippers committed:
fix: incorrect upgrade command for package managers (#2491)

vasfvitor committed:
[i18n-ignore] fix zh-cn plugin add cli usage (#2498)

renovate[bot] committed:
fix(deps): update dependency typedoc-plugin-mdn-links to v3.2.7 (#2503)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

GuoJikun committed:
docs: add `Migrate from Tauri 2.0 beta` in `/start/migrate/index.mdx` (#2504)

patelka2211 committed:
Improvements in nextConfig's type (#2502)

FabianLars committed:
docs: Fix global-shortcut rust example

tweidinger committed:
Explain wen "typo" (#2511)

xiaonerner0821 committed:
Update create-project.mdx (#2509)

renovate[bot] committed:
chore(deps): update packages/awesome-tauri digest to d93de19 (#2496)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Paul Valladares <85648028+dreyfus92@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency astro to v4.13.2 (#2513)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Fabian-Lars <fabianlars@fabianlars.de>

FabianLars committed:
ci: Fix token permissions (#2514)

FabianLars committed:
ci: Fix event_name comparison

GuoJikun committed:
i18n(zh-cn): Translate `start/migrate/from-tauri-2-beta.mdx` (#2505)

Co-authored-by: DK Liao <dklassic@gmail.com>

GuoJikun committed:
i18n(zh-cn): Synchronizing content (#2512)

KikkiZ committed:
i18n(zh-cn): Update the translation of develop/Plugins/index.mdx (#2391)

Co-authored-by: DK Liao <dklassic@gmail.com>

FabianLars committed:
ci: Use manual `gh api` request to assign users.

The `gh` cli does not contain pr users that are not part of the org assignable users.

homersimpsons committed:
:memo: Update haptics and geolocation plugins link (#2526)

renovate[bot] committed:
fix(deps): update dependency @types/node to v20.14.15 (#2527)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

vasfvitor committed:
feat: Plugin permissions component (#1890)

Co-authored-by: Paul Valladares <85648028+dreyfus92@users.noreply.github.com>
Co-authored-by: Lucas Nogueira <lucas@tauri.app>

lucasfernog committed:
fix(plugins): permission introduction consistency

renovate[bot] committed:
fix(deps): update dependency astro to v4.13.3 (#2528)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

ndom91 committed:
docs: update CI template for GHA (#2532)

renovate[bot] committed:
fix(deps): update dependency typedoc-plugin-mdn-links to v3.2.8 (#2533)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/awesome-tauri digest to c1f4fe2 (#2529)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Paul Valladares <85648028+dreyfus92@users.noreply.github.com>

lucasfernog committed:
[i18n-ignore] fix(plugins): preserve permission anchor links (#2530)

lucasfernog committed:
feat(plugins): finish file system documentation (#2531)

lucasfernog committed:
feat: expand calling-rust section (#2499)

lucasfernog committed:
feat: finish logging documentation (#2521)

lucasfernog committed:
feat: finish updater documentation (#2520)

lucasfernog committed:
feat: autogenerate CLI command docs (#2535)

naman-crabnebula committed:
docs: Add Biometric Plugin documentation (#2395)

Co-authored-by: Lucas Nogueira <lucas@tauri.studio>

renovate[bot] committed:
chore(deps): update packages/plugins-workspace digest to 78061c3 (#2493)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

lucasfernog committed:
Merge remote-tracking branch 'origin/v2' into notification-docs

tweidinger committed:
Blog: Update Audit References (#2538)

renovate[bot] committed:
fix(deps): update dependency starlight-blog to ^0.12.0 (#2540)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/plugins-workspace digest to 5081f30 (#2541)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency astro to v4.13.4 (#2543)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency shiki to v1.13.0 (#2544)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency astro to v4.14.0 (#2545)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update pnpm to v9.7.1 (#2510)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Paul Valladares <85648028+dreyfus92@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/awesome-tauri digest to f05e1ef (#2549)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/plugins-workspace digest to 4dd7f5d (#2550)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency @types/node to v20.15.0 (#2548)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency astro to v4.14.2 (#2546)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/plugins-workspace digest to 121f68b (#2552)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency @tauri-apps/cli to v2.0.0-rc.4 (#2553)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency shiki to v1.14.1 (#2554)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency typedoc-plugin-mdn-links to v3.2.9 (#2555)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency @types/node to v20.16.0 (#2557)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/plugins-workspace digest to 6c823ae (#2556)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Paul Valladares <85648028+dreyfus92@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency sharp to v0.33.5 (#2547)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Paul Valladares <85648028+dreyfus92@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency @types/node to v20.16.1 (#2559)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

jqpe committed:
docs: fix typo in `Plugins/index` (#2561)

renovate[bot] committed:
fix(deps): update dependency @tauri-apps/cli to v2.0.0-rc.5 (#2562)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

cleveng committed:
docs(zh-cn): fix dialog api typos (#2566)

GuoJikun committed:
docs: Fix store example (#2563)

renovate[bot] committed:
chore(deps): update packages/tauri digest to 3beba92 (#2466)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Paul Valladares <85648028+dreyfus92@users.noreply.github.com>

GuoJikun committed:
docs: Clearify tray positioner example (#2564)

timuric committed:
docs: Add new `core:` prefix to window capabilities (#2537)

renovate[bot] committed:
fix(deps): update dependency @tauri-apps/cli to v2.0.0-rc.6 (#2569)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: FabianLars <fabianlars@fabianlars.de>

renovate[bot] committed:
fix(deps): update dependency astro to v4.14.3 (#2568)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

GuoJikun committed:
i18n(zh-cn): Adjust menu order and sync content (#2518)

Co-authored-by: Fabian-Lars <fabianlars@fabianlars.de>

naman-crabnebula committed:
Merge branch 'tauri-apps:v2' into notification-docs

renovate[bot] committed:
fix(deps): update dependency astro to v4.14.4 (#2571)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

jonathanyeong committed:
Add trailing slash to state management url (#2577)

renovate[bot] committed:
chore(deps): update packages/awesome-tauri digest to 81a2050 (#2581)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

cleveng committed:
fix: fix api import (#2572)

GuoJikun committed:
docs: fix api.js import (#2591)

renovate[bot] committed:
fix(deps): update dependency jsdom to v24.1.3 (#2587)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency typedoc-plugin-mdn-links to v3.2.10 (#2586)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

RoseBlume committed:
Update Distribute Overview (#2576)

Co-authored-by: Lucas Fernandes Nogueira <lucas@tauri.app>
Co-authored-by: Tillmann <28728469+tweidinger@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency jsdom to v24.1.3 (#2593)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency jsdom to v25 (#2594)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

b6k-dev committed:
[i18n-ignore] fix: add missing dev:setup command to getting started guide (#2598)

lucasfernog committed:
feat: plugin setup docs consistency, fix desktop & mobile only setup (#2597)

lucasfernog committed:
feat: add signing docs (#2579)

* move all signing docs to its own menu under distribute
* added signing for macOS, iOS, Windows, Linux
* cleaned up the android docs to align with the other ones
* added signing to distribute overview

Niavana97 committed:
i18n(zh-cn): Fix /plugin/barcode-scanner.mdx (#2599)

b6k-dev committed:
docs: fix log target configuration examples (#2596)

lucasfernog committed:
feat: app bundle, DMG and App Store (macOS & iOS) distribution docs (#2583)

Co-authored-by: Tillmann <28728469+tweidinger@users.noreply.github.com>

Sir-Thom committed:
docs:fix the caution  in distribute/signing/linux  (#2601)

lucasfernog committed:
feat: add Windows installer and Microsoft Store distribution guides (#2589)

Co-authored-by: Tillmann <28728469+tweidinger@users.noreply.github.com>

lucasfernog committed:
feat: add Google Play distribution guide (#2588)

Co-authored-by: Tillmann <28728469+tweidinger@users.noreply.github.com>

lucasfernog committed:
fix: tauri paths (#2611)

GuoJikun committed:
Fix formatting errors (#2607)

RalfKit committed:
[i18n-ignore] fix: trailing slash across docs (#2605)

ahkohd committed:
feat: add collapse to the table of content items (#2519)

simonhyll committed:
fix: only update if it's still open

renovate[bot] committed:
fix(deps): update dependency astro to v4.15.1 (#2575)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

simonhyll committed:
fix: only update if it's still open

RalfKit committed:
[i18n-ignore] fix: missing slashes in href elements (#2616)

tweidinger committed:
fix(clipboard-manager): Update rust usage (#2609)

vjousse committed:
fix: flatpak runtimes names (#2610)

vasfvitor committed:
minor dialog doc fix (#2617)

Co-authored-by: Paul Valladares <85648028+dreyfus92@users.noreply.github.com>

lucasfernog committed:
fix: rename starlight-toc component (#2622)

vasfvitor committed:
register TAURI_WINDOWS_SIGNTOOL_PATH env var (#2618)

Co-authored-by: Paul Valladares <85648028+dreyfus92@users.noreply.github.com>

lucasfernog committed:
fix: macos signing

renovate[bot] committed:
fix(deps): update dependency @tauri-apps/cli to v2.0.0-rc.10 (#2574)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update packages/tauri digest to c55474f (#2625)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

lucasfernog committed:
feat: PermissionState is now exported from tauri (#2627)

renovate[bot] committed:
fix(deps): update dependency typedoc-plugin-mdn-links to v3.2.11 (#2626)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update dependency simple-git to v3.26.0 (#2628)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

knownasnaffy committed:
[Docs]: Update window customization plugin guide (#2624)

TACIXAT committed:
Specify that invoke is in window.__TAURI__.core (#2613)

tweidinger committed:
fix: Clarify Plugin Naming Convention (#2623)

tillmann-crabnebula committed:
chore: Prettier Fix (#2629)

renovate[bot] committed:
fix(deps): update dependency shiki to v1.16.1 (#2619)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency astro to v4.15.2 (#2615)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency astro to v4.15.2 (#2632)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

simonwong committed:
docs: fix some fn run miss pub (#2600)

renovate[bot] committed:
chore(deps): update packages/plugins-workspace digest to 51819c6 (#2621)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

NikolaRHristov committed:
Insert correct key (#2630)

Co-authored-by: vasfvitor <gitkey@virtuaires.com.br>

renovate[bot] committed:
chore(deps): update dependency starlight-links-validator to ^0.11.0 (#2614)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: vasfvitor <gitkey@virtuaires.com.br>

renovate[bot] committed:
fix(deps): update dependency @types/node to v20.16.3 (#2603)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency shiki to v1.16.1 (#2631)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update dependency simple-git to v3.26.0 (#2634)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency typedoc-plugin-mdn-links to v3.2.11 (#2633)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
fix(deps): update dependency astro to v4.15.2 (#2635)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

KirillTregubov committed:
docs: fix Distribute docs platform links (#2636)

swordtraveller committed:
i18n(zh-cn): Translation /develop/Debug/rustrover.mdx (#2585)

GuoJikun committed:
i18n(zh-cn): Translate `distribute/Signing/*` (#2608)

Co-authored-by: DK Liao <dklassic@gmail.com>

tweidinger committed:
Merge branch 'v2' into notification-docs
